setup/www/resources/config/benchmarking.nodejs.org

```diff
@@ -0,0 +1,38 @@
+server {
+    listen *:80;
+    server_name benchmarking.nodejs.org;
+
+    return 301 https://benchmarking.nodejs.org$request_uri;
+}
+
+server {
+    listen [::]:443 ssl http2;
+    listen *:443 ssl http2;
+    server_name benchmarking.nodejs.org;
+
+    ssl_certificate ssl/nodejs_chained.crt;
+    ssl_certificate_key ssl/nodejs.key;
+    ssl_trusted_certificate ssl/nodejs_chained.crt;
+
+    keepalive_timeout 60;
+    server_tokens off;
+
+    resolver 8.8.4.4 8.8.8.8 valid=300s;
+    resolver_timeout 10s;
+
+    add_header Strict-Transport-Security max-age=63072000;
+    add_header X-Frame-Options DENY;
+    add_header X-Content-Type-Options nosniff;
+
+    access_log /var/log/nginx/benchmarking/benchmarking.nodejs.org-access.log nodejs;
+    error_log /var/log/nginx/benchmarking/benchmarking.nodejs.org-error.log;
+
+    gzip on;
+    gzip_static on;
+    gzip_disable "MSIE [1-6]\.";
+    gzip_types text/plain text/css application/x-javascript text/xml application/xml application/xml+rss text/javascript;
```

Comment:
Oh, and I think the two javascript mimes can be reduced to just `application/javascript`.